docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -23,23 +18,34 @@ Traces can be discovered by searching logs for entries containing trace IDs.  Th
 
 
 ## Tempo search
-<span style="background-color:#f3f973;">This experimental feature is disabled by default. See below for more information on how to enable.</span>
+<span style="background-color:#f3f973;">Tempo search is an experimental feature.</span>
+
+### Search of recent traces
 
-Tempo includes the ability to search recent traces.  Traces can be searched for data originating from a specific service, duration range, and span and process-level attributes included in your application's instrumentation, such as HTTP status code, customer ID, etc.  Currently only search of traces at the ingesters is supported. By default the ingesters store the last 15 minutes.
+Tempo includes the ability to search recent traces held in ingesters.
+Traces can be searched for data originating from a specific service,
+duration range, and span and process-level attributes included in your application's instrumentation such as HTTP status code and customer ID.
```

Comment:
In this case span and process are two kinds of attributes, not sure they should be separated by the semicolon.

Reply:
I'll update the docs, since span and process-level attrs are two items in this list.  Given the odd composition of the original sentence, I didn't realize that.